src/main/flight/pid.c

```diff
@@ -281,6 +284,11 @@ void pidInitFilters(const pidProfile_t *pidProfile)
     }
 
     pt1FilterInit(&throttleLpf, pt1FilterGain(pidProfile->throttle_boost_cutoff, dT));
+    if (itermRelax)     
+        for (int i = 0; i < 3; i++) {
```

Comment:
`XYZ_AXIS_COUNT` instead of '3' here. we typically use:
`for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {`

Reply:
Ok. In case we ever decide to change the value of XYZ_AXIS_COUNT? BTW some code also does for (int axis = ROLL; ...) which I actually find worse than starting from 0 since it assumes the numbering of the axis.

Reply:
Oh just trying to make this fix I notice that XYZ_AXIS_COUNT is used very inconsistently. Should I just change it everywhere? It does show that 3 is used because of the number of axis which can be helpful.